src/test/java/org/reaktivity/nukleus/http_cache/internal/streams/server/ServerIT.java

```diff
@@ -27,6 +28,7 @@
 import org.kaazing.k3po.junit.rules.K3poRule;
 import org.reaktivity.reaktor.test.ReaktorRule;
 
+@Ignore
```

Comment:
Why `@Ignore` ?

Reply:
mistake, fixing! thanks  I think that was the missing 2% test coverage